operations/patch_util.go

```diff
@@ -242,6 +244,10 @@ func (p *patchParams) validatePatchCommand(ctx context.Context, conf *ClientSett
 		grip.Warningf("warning - failed to set default parameters: %s\n", err)
 	}
 
+	if p.Uncommitted && p.Ref != head {
```

Comment:
conf.UncommittedChanges is relevant here too, right, since the behavior is also silently ignored in this case? Should this just be an if statement within the one below?

Reply:
if so should this case also be added to the test?